### PR TITLE
[EWPP-1360] EWPP-1363: Add Sources field.

### PR DESCRIPTION
**Pull request by 22Alexandra**
### Description

Adds Sources field.

### Change log

- Added: Source field
- Changed: News form display.

### CODE REVIEW

File: modules/oe_content_news/config/install/field.field.node.oe_news.oe_news_sources.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  module:
+    - link
+id: node.oe_news.oe_news_sources
+field_name: oe_news_sources
+entity_type: node
+bundle: oe_news
+label: Sources
+description: ''
+required: false
+translatable: true
```
Comment (sergepavle) [inline modules/oe_content_news/config/install/field.field.node.oe_news.oe_news_sources.yml]: We have to indicate the translatability of the field only at the ewcms level.

File: modules/oe_content_news/config/install/field.storage.node.oe_news_sources.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    - node
+id: node.oe_news_sources
+field_name: oe_news_sources
+entity_type: node
+type: link
+settings: {  }
+module: link
+locked: false
+cardinality: -1
+translatable: true
```
Comment (sergepavle) [inline modules/oe_content_news/config/install/field.storage.node.oe_news_sources.yml:15]: We have to indicate the translatability of the field only at the ewcms level.

File: modules/oe_content_news/config/post_updates/00007_create_fields/field.field.node.oe_news.oe_news_sources.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  module:
+    - link
+id: node.oe_news.oe_news_sources
+field_name: oe_news_sources
+entity_type: node
+bundle: oe_news
+label: Sources
+description: ''
+required: false
+translatable: true
```
Comment (sergepavle) [inline modules/oe_content_news/config/post_updates/00007_create_fields/field.field.node.oe_news.oe_news_sources.yml]: We have to indicate the translatability of the field only at the ewcms level.

File: modules/oe_content_news/config/post_updates/00007_create_fields/field.storage.node.oe_news_sources.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    - node
+id: node.oe_news_sources
+field_name: oe_news_sources
+entity_type: node
+type: link
+settings: {  }
+module: link
+locked: false
+cardinality: -1
+translatable: true
```
Comment (sergepavle) [inline modules/oe_content_news/config/post_updates/00007_create_fields/field.storage.node.oe_news_sources.yml:15]: We have to indicate the translatability of the field only at the ewcms level.

**Review by sergepavle — CHANGES_REQUESTED** [review]

File: modules/oe_content_news/config/install/field.storage.node.oe_news_sources.yml (hunk 2 above)
Comment (22Alexandra) [reply modules/oe_content_news/config/install/field.storage.node.oe_news_sources.yml:15]: The storage needs to be translatable in order to be able to set the field translatable when needed (ewcms).

File: modules/oe_content_news/config/post_updates/00007_create_fields/field.storage.node.oe_news_sources.yml (hunk 4 above)
Comment (22Alexandra) [reply modules/oe_content_news/config/post_updates/00007_create_fields/field.storage.node.oe_news_sources.yml:15]: The storage needs to be translatable in order to be able to set the field translatable when needed (ewcms).

**Review by sergepavle — DISMISSED** [review]

**Review by nagyad — DISMISSED** [review]

**Review by sergepavle — APPROVED** [review]

**Review by nagyad — APPROVED** [review]

**Pull request merged**